### PR TITLE
chore(core): normalize autoloading from plugin scripts without warnings

### LINKED ISSUE #8982: Warnings caused by optional include()
For sniffing the location of autoload.php, we shouldn't use this pattern as it raises warnings:
```php
if (!include $path) {
    // try another path
}
```

### PR DESCRIPTION
**Pull request by mrclay**
Fixes #8982

### CODE REVIEW

**Comment by juho-jaakkola** [comment]
LGTM

File: mod/file/download.php [hunk 1]
```diff
@@ -4,7 +4,12 @@
  * 
  * @package ElggFile
  */
-require_once __DIR__ . '/../../vendor/autoload.php';
```
Comment (beck24) [inline mod/file/download.php]: What's wrong with this pattern?
Putting an is_file() check in there will just create fatal errors further down if there is no file

Comment (beck24) [inline mod/file/download.php]: Oh nevermind, I misread what it was doing - I thought you were skipping the require of the file if it didn't exist

**Comment by beck24** [comment]
LGTM

**Pull request merged**